src/Dfe.PlanTech.Web/Views/Recommendations/RecommendationContent.cshtml

```diff
@@ -21,7 +23,16 @@
                             await Html.RenderPartialAsync("Components/PageComponentFactory", content);
                         }
                     </div>
+                    @if (headerWithContent is RecommendationChunk chunk && !string.IsNullOrEmpty(chunk.CSUrl) && !string.IsNullOrEmpty(chunk.CSLinkText))
+                    {
+                        <div class="govuk-!-margin-top-3 govuk-!-margin-bottom-9">
```

Comment:
Bottom margin of 9 seems slightly too much but I'm not sure exactly what we want to go for, prototype indicates using top/bottom of 4 or 5 so I'd go with one of those and we can double check it